src/interceptors.ts

```diff
@@ -0,0 +1,13 @@
+
+export const AxleTIMEOUTInterceptor = () => {
```

Comment:
rename to timeoutInterceptor and support options.normalizeErrorCode and defaults 'TIMEOUT'

Reply:
I think maybe `normalizeErrorCode` option is probably less necessary